spec/models/user_spec.rb

```diff
@@ -941,4 +941,56 @@ def user_from_authorization_service(service_name, signed_in_resource, cta_varian
       expect(user.receives_follower_email_notifications?).to be(true)
     end
   end
+
+  describe "#number_visits_per_day" do
+    it "returns the number of visits per day" do
+      user.sign_in_count = 8
+      user.created_at = Time.zone.today.days_ago(4)
+      expect(user.number_visits_per_day).to eq(2.0)
+      user.created_at = Time.zone.today
+      expect(user.number_visits_per_day).to eq(8.0)
+    end
+  end
+
+  describe "#number_articles_read" do
+    it "returns the number of articles read in total" do
+      create_list(:page_view, 10, user: user)
```

Comment:
creates 10 page views with factory bot with their associated articles for the current user